docs/influence-sensor-execution-order.md

```diff
@@ -0,0 +1,73 @@
+# Influencing the Sensors execution order
+
+## Preamble
+
+The following document applies for 3 kinds of classes in plugins, the ones implementing:
+* `org.sonar.api.batch.sensor.Sensor`
+* `org.sonar.api.scanner.sensor.ProjectSensor`
+* `org.sonar.api.batch.postjob.PostJob`
```

Comment:
Either don't mention `PostJob`, or change the title to `Influencing certain extensions points execution order`

Reply:
Yes, I think it would be better to talk about extensions and extension points in this document. I will give another try